frontend/src/machines/billings/billingMachine.ts

```diff
@@ -6,6 +6,10 @@ import {
   BillingDetail,
   BillingMember,
 } from "../../utils/fetcher";
+import { QueryClient } from "react-query";
+import { getBillings, getMembers } from "./mockFetcher";
```

Comment:
ini bisa langsung ambil dari `../../utils/fetcher` aja, gak perlu bikin function sendiri kayaknya ?

Reply:
Semacam ini :

```
import { membersAPI } from '../../utils/fetcher'

// nama functionnya berdasarkan "operationId" di api.json
membersApi.getMemberList() 
membersApi.getBilling() 
```

Reply:
Oalah, bisa ya? 
Ku kira karena BE nya belum ready jadinya belum bisa

Reply:
Ini udah aku coba ya, dan gabisa. Data nya ga nampil dan fetch nya failed

Reply:
host nya diperbaiki dulu, itu masih ngarah ke `localhost:4000`

Reply:
gantinya di `utils/fetcher.ts` ya